src/main/kotlin/net/atos/zac/identity/IdentityService.kt

```diff
@@ -60,4 +67,22 @@ class IdentityService @Inject constructor(
             .map { it.toUser() }
             .sortedBy { it.getFullName() }
     }
+
+    fun checkIfUserIsInGroup(userId: String?, groupId: String?) {
```

Comment:
Is this really needed I was wondering? Can't we just do a more clever call to Keycloak when we retrieve the user by doing a 'get user from group' call if that is possible?

Reply:
At least we could start using the Keycloak `GET /admin/realms/{realm}/users/{user-id}/groups` API endpoint I think? I.e. a `listGroupsForUser` call. I think for this scenario that will be more efficient and makes more sense than the other way around (`listUsersInGroup`). Also because in future the `listUsersInGroup` may well return hundreds if not more users..

Reply:
Now listing users

Reply:
nice!